src/seismometer/data/pandas_helpers.py

```diff
@@ -543,9 +535,11 @@ def event_value_name(event_value: str) -> str:
     return val
 
 
-def valid_event(dataframe: pd.DataFrame, event: str) -> pd.DataFrame:
-    """Filters a dataframe to valid predictions, where the event value has not set to -1."""
-    return dataframe[dataframe[event_value(event)] >= 0]
+def is_valid_event(dataframe: pd.DataFrame, event: str, ref: str) -> pd.DataFrame:
```

Comment:
Should is_valid_event include the event's merge_strategy?

Reply:
I am curious if we ever want a model to predict events in the past? - For example, we may want to check if an event in the past was erroneously added to the chart (say a DX code was added, but was incorrect, or the Observation status assigned to a patient was incorrect)

Reply:
Currently we don't have any validation/restriction on merge_strategy so isnt needed (yet).

In your scenario, I don't know think that predicttime is relevant.  And this kind of supports my implementation of "default True" keeping everything.
Unless it's predicting the event+time, I think we'd want to make sure we could strip times since they aren't relevant.